wiki/Tournaments/Official_support/de.md

```diff
@@ -81,9 +79,9 @@ Zusätzlich sollten alle Werbematerialien oder andere Dienstleistungen, die im Z
 - Alle registrierten Teilnehmer müssen vor Spielbeginn vom Account-Support-Team überprüft werden.[^play-commence] Siehe [Turnier-Screening](#turnier-screening) für mehr Details.
 - Wenn ein Nutzer die Kriterien zum Registrieren erfüllt (falls vorhanden), darf das Turnier keinen an der Teilnahme hindern, der das Screening-Verfahren besteht, außer es werden zahlreiche Beweise gegen den Teilnehmer öffentlich vorgelegt und das Account-Support-Team stimmt zu.
   - Dazu gehört, dass Spieler, die als absichtliche Verlierer wahrgenommen werden (*Sandbagging*), vom Spiel ausgeschlossen werden. Sollten Veranstalter berechtigte Bedenken über die Anwesenheit solcher Spieler haben, die die Wettbewerbsintegrität des Turniers beeinflussen, können sie das Problem mit Hilfe des [Meldeformulars für Turniere](https://pif.ephemeral.ink/tournament-reports) an das [Turnierkomitee](/wiki/People/Tournament_Committee) für eine Einzelfallüberprüfung melden.
-- Ein engagierter Schiedsrichter muss bei jedem Match präsent sein. Spieler können nicht selbst Schiedsrichter sein.
+- Ein fest zugeordneter Schiedsrichter muss bei jedem Match anwesend sein oder innerhalb eines minimalen Zeitraums zur Verfügung stehen (2 bis 5 Minuten), wenn eine entsprechende Genehmigung für automatisierte Software eingeholt wurde und zur Unterstützung eingesetzt wird. Spieler können nicht selbst Schiedsrichter sein.
   - Der Einsatz von Bots ist für die Qualifizierungsphase gestattet, sofern dies von einem Mitarbeiter beaufsichtigt wird.
-- Asynchrone Spiele sind unter keinen Umständen erlaubt. Alle teilnehmenden Teams müssen zur geplanten Spielzeit anwesend sein.
+- Jedes Spiel innerhalb derselben Gruppen- oder Qualifikationsrunde sollte in einem einheitlichen Format sein. In einem Turnier, das zum Beispiel eine Qualifikationsphase und ein Doppeleliminierungsverfahren verwendet, sollten alle Entscheidungsspiele entweder synchron oder asynchron gespielt werden, aber nicht beides.
```

Comment:
- `bracket`: Normally, I would have translated this with `Tunierbaum`. In the case above, this didn't make sense to me, so I used `Gruppe(nphase/runde)`.
- `bracket match`: I'm not entirely sure if this has some special meaning compared to just `match`. I translated it to `Entscheidungsspiel`.